image_provisioner/playbooks/roles/pbench-config/tasks/main.yaml

```diff
@@ -1,37 +1,49 @@
 ---
-- name: rename pbench config
-  command: mv /opt/pbench-agent/config/pbench-agent.cfg.example /opt/pbench-agent/config/pbench-agent.cfg
-  args:
-    creates: /opt/pbench-agent/config/pbench-agent.cfg
+- block: 
+   - name: rename pbench config
+     command: mv /opt/pbench-agent/config/pbench-agent.cfg.example /opt/pbench-agent/config/pbench-agent.cfg
+     args:
+       creates: /opt/pbench-agent/config/pbench-agent.cfg
 
-- name: configure pbench target
-  replace:
-    dest: /opt/pbench-agent/config/pbench-agent.cfg
-    regexp: '{{ pbench_internal_hostname }}'
-    replace: '{{ pbench_alternate_hostname }}'
-    backup: yes
+   - name: configure pbench target
+     replace:
+       dest: /opt/pbench-agent/config/pbench-agent.cfg
+       regexp: '{{ pbench_internal_hostname }}'
+       replace: '{{ pbench_alternate_hostname }}'
+       backup: yes
 
-- name: copy ssh key to pbench directory
-  copy:
-    src: /root/svt-private/image_provisioner/id_rsa 
-    remote_src: True
-    dest: /opt/pbench-agent/id_rsa
-    owner: pbench
-    group: pbench
-    mode: 0600
+   - name: copy ssh key to pbench directory
+     copy:
+       src: /root/svt-private/image_provisioner/id_rsa 
+       remote_src: True
```

Comment:
we don't want remote_src to be set to true since for atomic, the repos are cloned on the localhost rather than on the remote.